code/modules/mob/living/carbon/carbon.dm

```diff
@@ -624,6 +624,13 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 
 	dat += "<tr><td><B>Mask:</B></td><td><A href='?src=[UID()];item=[slot_wear_mask]'>[(wear_mask && !(wear_mask.flags&ABSTRACT)) ? wear_mask : "<font color=grey>Empty</font>"]</A></td></tr>"
 
+	if(istype(wear_mask, /obj/item/clothing/mask/muzzle))
+		var/obj/item/clothing/mask/muzzle/M = wear_mask
+		if(M.security_lock)
+			dat += "&nbsp;<A href='?src=[UID()];locked=[slot_wear_mask]'>[M.locked ? "Disable Lock" : "Set Lock"]</A>"
```

Comment:
I did a copy of the internals code, I am not sure exactly what you mean by using `?src=[M.UID()]`